packages/core/src/helpers/address.ts

```diff
@@ -0,0 +1,17 @@
+import { utils } from 'ethers'
+import { BigNumber } from '@ethersproject/bignumber'
+
+export function shortenAddress(address: string) {
```

Comment:
Considerations:
1) Add validation (throw an error if not a valid address)
2) Add second helper (next PR) `shortenIfAddress(address: Falsy<string>)` which will shorten address or return empty string if falsy value